src/styles/components/_title.scss

```diff
@@ -7,9 +7,16 @@
 .homepage_section_block {
 	align-items: center;
     height: 450px;
+<<<<<<< HEAD
```

Comment:
please fix the merge conflict here?

Reply:
that doesn't appear anymore when I redone the previous PR. what should I do?

Reply:
also don't know what I am messing as didn't merge anything and it merged and closed the PR :(